bundler/lib/bundler/checksum.rb

```diff
@@ -179,16 +185,14 @@ def inspect
       # that contain the same gem with different checksums.
       def replace(spec, checksum)
         return unless checksum
-
```

Comment:
We do generally keep a blank line after guards, I think.

Reply:
Yeah, this is a bit claustrophobic. Will fix.